frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -101,19 +102,46 @@ class HtmlDocument extends React.Component {
         keepEditingWarning: true
       })
     }
+    // TODO test and review
+    let commentList = state.timeline.filter(item => item.timelineType === 'comment')
+    let {
+      actives_shares,
+      author,
+      created,
+      current_revision_id,
+      current_revision_type,
+      last_modifier,
+      ...revisionObject
+    } = data.content
 
     this.setState(prev => ({
       ...prev,
       content: {
         ...prev.content,
         ...data.content
-        // raw_content: data.raw_content // TODO blocked by back
       },
       newComment: localStorageComment || '',
-      rawContentBeforeEdit: prev.content.raw_content
+      rawContentBeforeEdit: prev.content.raw_content,
+      timeline: [
+        ...prev.timeline,
+        {
+          ...revisionObject,
+          author: {
+            public_name: data.author.public_name,
+            avatar_url: data.author.avatar_url,
+            user_id: data.author.user_id
+          },
+          commentList: commentList,
```

Comment:
I think commentList is the entire comments list of this content.
And I think, here, you are associating all these comments to the new revision
(I'm not sure I correct here)